src/Termonad/Config.hs

```diff
@@ -62,4 +64,5 @@ defaultTMConfig =
     , cursorColor = lightgrey
     , scrollbackLen = 10000
     , confirmExit = True
+    , wordCharExceptions = ":./"
```

Comment:
Could you change this to be something like the following:

`"-#%&+,./=?@\\_~\183:"`

This should be the default list of wordchar exceptions, plus `:`.

(As an aside, I was able to figure out the default list of wordchar exceptions by checking out what the output of [terminalGetWordCharExceptions](http://hackage.haskell.org/package/gi-vte-2.91.19/docs/GI-Vte-Objects-Terminal.html#g:40) is.)

That way, everything that is currently highlightable will continue to be highlightable, plus URLs.

Without this, things like email addresses `user@gmail.com` are no longer highlightable in one double-click.